### ISSUE TITLE
Implement stock "Warp To" function

### ISSUE THREAD

**Issue by hvacengi** [KSP-KOS/KOS#1029]
Implement access to the stock "Warp To" function.

This is pretty easy to do, but has one significant limitation: you cannot cancel "Warp To" without keyboard interaction.  As long as we make that limitation explicitly clear to the user, that should be acceptable.

Currently, I'm implementing this as a function (in Misc.cs) which accepts 1 to 3 parameters (required number of parameters is shown in parenthesis): 
ut: the time to warp to (1-2-3)
maxwarp: the maximum warp setting to use (2-3)
minwarp: the minimum warp setting to use (3)

I don't really know what the minwarp setting does, but it is a stock option so I wanted to expose it.

**Comment by evg-zhabotinsky** [comment]
Well, I have just written my own timedwarp.ks and run it whenever I need:
```
parameter warp_end.
local lock tm to warp_end - time:seconds.
print "Warping " + tm + " seconds forward.".
set warp to 0.
if tm > 5 {
	set warp to 1.
	if tm > 10 {
		set warp to 2.
		// And so on, nested further for higher warp speeds
		wait until tm < 10.
		set warp to 1.
	}
	wait until tm < 5.
	set warp to 0.
}
wait until tm <= 0.
print "Timed warp done.".
```

It takes 1 parameter (moment to warp to) and decreases warp speed 1 second (real time) before destination (because it takes some time to stop warp).
Only problem: it fails when you run out of power :)

------------

**On subject:**
To overcome "can't stop" problem of KSP's "Warp to" function, you can implement it yourself.
Basically the above logic, invoked with a function call and running in background, but also stop on writes to WARP variable.
